include/seqan3/std/span

```diff
@@ -70,12 +70,23 @@ inline constexpr bool enable_safe_range;
 template<class span_tp>
     requires std::is_span<span_tp>::value
 inline constexpr bool enable_safe_range<span_tp> = true;
+
+// ranges library defines it as, there as we allow const and reference versions:
```

Comment:
There is something wrong in this sentence with `it as, there as`

Reply:
This was fixed in #1712